app/models/sample.rb

```diff
@@ -144,6 +144,14 @@ def search_data
     }.compact
   end
 
+  def field?(field)
+    metadata.key?(field)
```

Comment:
This is great. There's a few spots in `/app/services/samples/metadata/update_service.rb` that uses `.metadata.key?`. Do you mind updating them to use `.field?`

Reply:
Updated in 8fb71c9ed298ebf44668f520ae5433f80caffdc3